modules/docs/arrow-docs/docs/docs/optics/syntax/README.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+title: Syntax DSL
+permalink: /docs/optics/syntax/
+---
+
+## Syntax DSL
+
+In some cases the full power of Optics is not required and a familiar property-like syntax to work with values of immutable structures is desired.
+To avoid boilerplate Arrow can generate this property-like dsl using `@syntax` annotation.
+
+```kotlin
+@syntax
```

Comment:
I like everything here but I think `syntax` is way too ambiguous since any of the modules could have named their annotations also syntax to expand exten funs for example. I think we should come up with a better one before the release but in whatever case this is beyond awesome!